packages/extension-browser/src/devtools/views/pages/config/config-label.css

```diff
@@ -1,4 +1,11 @@
 .root {
     display: block;
-    margin: 0.75rem 1.25rem;
+    font-size: 1.166em;
```

Comment:
Should this be in `rem` units? I don't think we're looking to make it relative to whatever container we get stuck in...